Formula/ghc.rb

```diff
@@ -44,6 +44,11 @@ class Ghc < Formula
   uses_from_macos "m4" => :build
   uses_from_macos "ncurses"
 
+  # Build uses sed -r option, which is not available in Catalina shipped sed.
+  on_catalina do
```

Comment:
Should this be on Catalina or older?

Reply:
Yes:
```
mojave$ /usr/bin/sed -h
/usr/bin/sed: illegal option -- h
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Reply:
The formula on [line 42](https://github.com/dkav/homebrew-core/blob/b53c7a77174f9ed448531e6d7b2e6a43b95b3dd9/Formula/ghc.rb#L42) limits a minimum of Catalina (see a46fa896dd9d474b165a9b99aeb61eecaf3df0cf) so I assumed that `older` was not required.

Reply:
Ok, fair enough.